tests/test-sources/plugins/pluginmanagers/lz-n.nix

```diff
@@ -0,0 +1,140 @@
+{ pkgs, helpers, ... }:
+let
+  inherit (pkgs) lib;
```

Comment:
This could be moved to "module" args in the respective test-cases to avoid needing to do `inherit (pkgs) lib`.

To do this you add a `module` attr in the test case which contains the actual test case's module function:

```nix
single-plugin = {
  module =
    {
      config,
      options,
      pkgs,
      lib,
      helpers,
      ...
    }:
    {
      # Test config module
    };
};
```